src/client/state/sagas/index.js

```diff
@@ -0,0 +1,182 @@
+import {
```

Comment:
We could break this file down into multiple sagas, I suppose..

Reply:
Yes, we could. I did that but I guess it got lost somewhere or reverted because git is awesome and I'm dumb. On it 👍